code-challenges.test.js

```diff
@@ -13,55 +18,161 @@
+//   array.push(array[i-1]) + array [i+2]
+//   return (array)
+// } 
+// (version 2) 
+const greaterThanTwo = (length) => {
+  let array = [1, 1];
+  for (let i = 2; i < length + 1; i++){
+    array.push(array[i - 2] + array[i -1])
+  }
+ return (array)
+}
+
+//  notes :  I'm still getting failures with this function, I know what I want to do I believe, but I don't know how to exactly express that in the code.  There are gaps in my logic that I'm just not knowledable enough to correct on my own yet. 
+
 
 // --------------------2) Create a function that takes in an object and returns an array of the values sorted from least to greatest.
 // Hint: Check out this resource: Object.values() https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values
 
 // a) Create a test with expect statements for each of the variables provided.
-
-const studyMinutesWeek1 = {
+// test outline :
+// describe('my test', () => {
+//   it('some text', () => {
+//       expect(...)...;
+//   })
+// });
+
+describe("leastToGreatest" , () => {
+     it( "returns an array of the values sorted from least to greatest ", () => {
+     expect(leastToGreatest(studyMinutesWeek1)).toEqual([15, 15, 20, 30, 30, 60, 90])
+     expect(leastToGreatest(studyMinutesWeek2)).toEqual( [10, 15, 20, 45, 60, 65, 100])
+   })
+   })
+ const studyMinutesWeek1 = {
   sunday: 90,
-  monday: 30,
+   monday: 30,
   tuesday: 20,
   wednesday: 15,
   thursday: 30,
   friday: 15,
-  saturday: 60
-}
-// Expected output: [15, 15, 20, 30, 30, 60, 90]
+  saturday: 60 }
+// // Expected output: [15, 15, 20, 30, 30, 60, 90]
 
 const studyMinutesWeek2 = {
-  sunday: 100,
-  monday: 10,
+ sunday: 100,
+ monday: 10,
   tuesday: 45,
   wednesday: 60,
   thursday: 20,
   friday: 15,
-  saturday: 65
+   saturday: 65
 }
 // Expected output: [10, 15, 20, 45, 60, 65, 100]
 
-// b) Create the function that makes the test pass.
+//output  ● leastToGreatest › returns an array of the values sorted from least to greatest 
 
-// --------------------3) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
+//     ReferenceError: leastToGreatest is not defined
 
-// a) Create a test with expect statements for each of the variables provided.
 
-const accountTransactions1 = [100, -17, -23, -9]
-// Expected output: [100, 83, 60, 51]
+// b) Create the function that makes the test pass.
+// psuedocode : Create a function called leastToGreatest that takes in an object and returns an array of the values sorted from least to greatest. The parameters for this function will be an array. The input will be an object and the expected output will be an array of values located within the array from least to greatest. use the JavaScript Object.values() method to access the own enumerable properties of an object. Object.values(obj)
+//  (version 3 )
+ const leastToGreatest = (studyMinutesWeek1,studyMinutesWeek2)  => {
```

Comment:
Again, really close here in the logic. I would suggest making the Object.values a declared variable that you can then sort in your function. So an example:
`const leastToGreatest = (obj) => {
  var thisObject = Object.values(obj)
  return thisObject.sort((a, b) => a-b))
  }`

Reply:
Thank you Nicole , this was helpful !